spec/introduction.md

```diff
@@ -1046,21 +1142,27 @@ public class List<T> {
     int count;
 
     // Constructors...
-    public List(int capacity = defaultCapacity) {
+    public List(int capacity = defaultCapacity)
+    {
         items = new T[capacity];
     }
 
     // Properties...
-    public int Count {
+    public int Count
+    {
         get { return count; }
     }
-    public int Capacity {
-        get {
+    public int Capacity
+    {
+        get
+        {
             return items.Length;
         }
```

Comment:
Inconsistent formatting here - the **get** for `Count`, above, is entirely on a single line; this one should be as well.